docs/html/user_guide.rst

```diff
@@ -469,20 +478,20 @@ pip's command line options can be set with environment variables using the
 format ``PIP_<UPPER_LONG_NAME>`` . Dashes (``-``) have to be replaced with
 underscores (``_``).
 
-For example, to set the default timeout::
-
-    export PIP_DEFAULT_TIMEOUT=60
-
-This is the same as passing the option to pip directly::
+For example, setting ``PIP_DEFAULT_TIMEOUT`` to 60 has the same effect as
+passing the option to pip directly::
 
     pip --default-timeout=60 [...]
 
-For command line options which can be repeated, use a space to separate
-multiple values. For example::
-
-    export PIP_FIND_LINKS="http://mirror1.example.com http://mirror2.example.com"
+For command line options which can be repeated, if they do not take any argument
+like ``--quite`` or ``--verbose``, use a non-negative integer to represent
+the number of repetition, for instance, setting ``PIP_VERBOSE`` to 3 is
+equivalent to calling pip with ``-vvv``.
 
-is the same as calling::
+Fruitful repeatable options, however, can be specified via environment variable
+as a space-separated list. For example, having ``PIP_FIND_LINKS`` set to
+``"http://mirror1.example.com http://mirror2.example.com"`` is the same as
+calling ::
 
```

Comment:
Again, let's leave the existing text as it is, and just add the extra text about options that don't take a value but can be repeated. (If we do want to change the text, though, "Fruitful repeatable options" doesn't mean anything to me - maybe it's a phrase that didn't translate well to English? Or maybe it's just something I've never heard before?)

Reply:
*Fruitful function* is used in [a book I read quite some time ago](https://www.greenteapress.com/thinkpython/thinkCSpy/html/chap05.html) to refer to function with a positive number of parameters but after looking it up all results point to this exact book (or its derivatives), so I think it's definitely not something to be used in a documentation and I am sorry for not checking it more carefully before hand<sup>*nervously thinking about everything I've written in the last few years*</sup>

What is your suggestion in this case to document the `count` and `append` options.  Those don't seem to be intuitive out of implementation context, and originally `count` options was not documented.

Reply:
>  I am sorry for not checking it more carefully before hand

Not a problem!

> What is your suggestion in this case

I would go with something like:

```suggestion

For command line options which can be repeated, use a space to separate
multiple values. For example::

    export PIP_FIND_LINKS="http://mirror1.example.com http://mirror2.example.com"

is the same as calling::

        pip install --find-links=http://mirror1.example.com --find-links=http://mirror2.example.com
        
Options that do not take a value, but can be repeated (such as ``--verbose``) can be specified using the number of repetitions, so::

    export PIP_VERBOSE=3

is the same as calling::

    pip install -vvv
```

I'm not sure if github's "Suggestion" thing handled this correctly, but I hope you see what I mean, leave the existing text alone and just add a new paragraph about count-style options.

Reply:
GitHub suggestion doesn't seem to work, but this is clearer than what I wrote earlier so I'll apply it manually.